frontend/src/modules/home/quiz/components/PlayButton.js

```diff
@@ -2,22 +2,25 @@ import { Button } from "@material-ui/core"
 import PlayArrowIcon from "@material-ui/icons/PlayArrow";
 import * as Tone from "tone";
 
-const PlayButton = ({first, second}) => {
-    const playTone = () => {
+
+const playTone = (first, second) => {
         //create a synth and connect it to the main output (your speakers)
         const synth = new Tone.Synth().toDestination();
         const now = Tone.now()
 
         //play a middle 'C' for the duration of an 8th note
```

Comment:
I noticed that this comment doesn't apply anymore. Maybe we can change it to something like
```suggestion
        // Play the first and second notes sequentially with the duration of an 8th note
```
Feel free to edit the comment if you think of something more fitting.